lib/cocoapods/installer/analyzer.rb

```diff
@@ -744,7 +744,7 @@ def generate_pod_target(target_definitions, target_inspections, specs, scope_suf
           archs = target_requires_64_bit ? ['$(ARCHS_STANDARD_64_BIT)'] : []
         end
         host_requires_frameworks = target_definitions.any?(&:uses_frameworks?)
-        platform = determine_platform(specs, target_definitions, host_requires_frameworks)
+        platform = determine_platform(specs.select(&:library_specification?), target_definitions, host_requires_frameworks)
```

Comment:
I would opt for putting this into `determine_platform` instead so the logic isn't spread out

Reply:
hm sure sg